cli/tests/common/fixtures.rs

```diff
@@ -1,304 +1,165 @@
-            "https://github.com/codekiln/langstar",
-            "--branch",
-            "main",
-            "--config-path",
-            "tests/fixtures/test-graph-deployment/langgraph.json",
-            "--deployment-type",
-            "dev",
-            "--wait",
-            "--format",
-            "json",
-        ]);
+        let result = runtime.block_on(async {
+            // Create SDK client
+            let auth = AuthConfig::from_env()
+                .expect("LANGSMITH_API_KEY and workspace ID must be set for integration tests");
+            let client =
+                langstar_sdk::LangchainClient::new(auth).expect("Failed to create LangchainClient");
 
-        let start = Instant::now();
-        let output = cmd.output().expect("Failed to execute deployment creation");
+            // Use shared get_or_create_deployment utility
+            get_or_create_deployment(&client, &config).await
+        });
 
-        if !output.status.success() {
-            panic!(
-                "Failed to create test deployment.\nStdout: {}\nStderr: {}",
-                String::from_utf8_lossy(&output.stdout),
-                String::from_utf8_lossy(&output.stderr)
-            );
-        }
+        let (deployment_id, _revision_id) = result.expect("Failed to get or create deployment");
 
-        let stdout = String::from_utf8_lossy(&output.stdout);
-
-        // Extract deployment ID from JSON output
-        let json_start = stdout
-            .find('{')
-            .expect("Should contain JSON object in output");
-        let json_str = &stdout[json_start..];
-        let json: serde_json::Value =
-            serde_json::from_str(json_str).expect("Should return valid JSON");
-        let deployment_id = json["id"]
-            .as_str()
-            .expect("Should have 'id' field")
-            .to_string();
-
-        let elapsed = start.elapsed();
-        println!("\n✅ Test deployment created successfully!");
-        println!("   Name: {}", deployment_name);
+        println!("\n=================================================");
+        println!("Test deployment ready");
+        println!("   Name: {}", config.name);
         println!("   ID: {}", deployment_id);
-        println!("   Creation time: {:.1}s", elapsed.as_secs_f32());
         println!("=================================================\n");
 
         Self {
             id: deployment_id,
-            name: deployment_name,
+            name: config.name,
         }
     }
 
+    /// Create a test deployment for release/lifecycle testing
+    ///
+    /// Uses the SDK's `for_release_tests()` config which creates a timestamped
+    /// deployment name ("release-integration-test-{timestamp}"). This ensures
+    /// a fresh deployment for testing the full create → test → delete lifecycle.
+    #[allow(dead_code)]
+    pub fn create_for_release() -> Self {
+        Self::create_with_config(TestDeploymentConfig::for_release_tests())
```

Comment:
I don't understand why this is dead code - or, conversely, if it is actually dead code, why we should keep it.